java-util/src/main/java/io/druid/java/util/common/RetryUtils.java

```diff
@@ -34,6 +33,16 @@
   public static final long MAX_SLEEP_MILLIS = 60000;
   public static final long BASE_SLEEP_MILLIS = 1000;
 
+  public interface Task<T>
```

Comment:
I don't see much value in replacing callable with this.

Reply:
> The Callable interface is similar to Runnable, in that both are designed for classes whose instances are potentially executed by another thread.

This is from the javadoc of Callable. I think it's better to read because Callable is typically used for multi-threading, and developers might mistakenly expect that the Callable is executed by another thread.